c++/src/kj/configure.bzl

```diff
@@ -76,10 +76,7 @@ def kj_configure():
 
     native.cc_library(
         name = "kj-defines",
-        defines = [
```

Comment:
let's make sure we add this to our build.

Reply:
Done in internal PR 4503.